File: python/1929.py
a, b, c, d = list(map(int, input().split(' ')))

if a + b > c and a + c > b and c + b > a:
    print("S")
elif a + b > d and a + d > b and b + d > a:
    print("S")
elif a+c > d and d + a > c and c + d > a:
    print("S")
elif b+c > d and b + d > c and c + d > b:
    print("S")
else:
    print("N")

File: js/PROBLEMS/1929.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split(' ')

a = parseInt(lines.shift())
b = parseInt(lines.shift())
c = parseInt(lines.shift())
d = parseInt(lines.shift())

if (a + b > c && a + c > b && c + b > a){
    console.log("S");
}else if (a + b > d && a + d > b && b + d > a) {
    console.log("S");
}else if (a+c > d && d + a > c && c + d > a){
    console.log("S");
}else if (b+c > d && b + d > c && c + d > b){
    console.log("S");
}else{
    console.log("N");
}